js/host/main.js

```diff
@@ -98,7 +98,32 @@ const nlwManager =
   new NLWManager( byEID("nlw-frame"), connMan.broadcast
                 , connMan.narrowcast, onNLWManError);
 
-document.addEventListener("DOMContentLoaded", nlwManager.init);
+document.addEventListener("DOMContentLoaded", () => {
+  const modalContainer = byEID("modal-container");
+
+  window.onclick = (event) => {
+    if (event.target === modalContainer) {
+      modalContainer.classList.toggle("hidden");
+    }
+  };
+
+  byEID("read-more-button").onclick = () => {
+    modalContainer.classList.toggle("hidden");
+  };
+
+  byEID("close-modal-button").onclick = () => {
+    modalContainer.classList.toggle("hidden");
+  };
+
+  document.addEventListener("keydown", (event) => {
+    console.log(modalContainer.className);
```

Comment:
Remove log line

Reply:
Ahhh gosh, totally missed this. Thanks for catching it!